incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -185,8 +186,15 @@ public final class ZillaDumpCommand extends ZillaCommand
     private final TcpHeaderFW.Builder tcpHeaderRW = new TcpHeaderFW.Builder();
     private final MutableDirectBuffer writeBuffer;
 
-    // visible for testing
-    Path directory;
+    private Path directory;
+    private BeginFW begin2RO = new BeginFW();
+    private DataFW data2RO = new DataFW();
+    private EndFW end2RO = new EndFW();
+    private AbortFW abort2RO = new AbortFW();
+    private ResetFW reset2RO = new ResetFW();
+    private FlushFW flush2RO = new FlushFW();
+    private ChallengeFW challenge2RO = new ChallengeFW();
+    private ExtensionFW extension = new ExtensionFW();
```

Comment:
These flyweights should be final and moved up as they are almost like non-static constants.

Reply:
Actually, these fields are no longer used and should be removed, after you define the return from the builders as local variables in each method below.

Reply:
removed as per the other comment